src/plugin.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "JSON",
-  "id": "json",
+  "id": "simpod-json-datasource",
```

Comment:
what about `universal datasource`?

Reply:
As you prefer. I'm happy with keeping your name in, its an internal id that will only show up during installation afaiks?

Reply:
Ok, we can leave it for it's more unique then